layouts/coverages.html

```diff
@@ -0,0 +1,9 @@
+<h1 id="coverage-workflow">Coverages</h1>
+
+<p>The Coverages resources are used to manage the lifecycle of patient insurance coverages within the healthcare setting.</p>
+
+<% if @item[:version] == 'dstu2' %>
```

Comment:
Why do we have a check for DSTU2? Isn't this for R4?

Reply:
we have this check in all layouts/xxx.html files that have both DSTU2 and R4 resources - we want to render this link for the DSTU2 landing page but not the R4